app/main.py

```diff
@@ -80,36 +79,49 @@ def display_page(pathname):
 # ------------------------------------------------------------------------------
 # Callbacks related to the "Alertes et Infrastructures" dashboard
 
+# Fetching reusable alert metadata
+alert_metadata = build_live_alerts_metadata()
```

Comment:
Isn't this contradictory with the fact that we are calling the `build_live_alerts_metadata` function in each of the functions that are related to alerts in the other files (notably `build_alerts_elements` or `build_alerts_markers` in `alerts.py` and `display_alerts_frames` in `homepage.py`)? Shouldn't these functions take an `alert_metadata` as argument so that we don't need to create it  each time we call these functions?